src/game_api/render_api.cpp

```diff
@@ -665,3 +686,51 @@ TextRenderingInfo::~TextRenderingInfo()
         game_free(unknown6);
     }
 }
+void TextRenderingInfo::rotate(float angle, std::optional<float> px, std::optional<float> py)
+{
+    const auto resx = static_cast<float>(get_setting(GAME_SETTING::RESOLUTIONX).value_or(16));
+    const auto resy = static_cast<float>(get_setting(GAME_SETTING::RESOLUTIONY).value_or(9));

```

Comment:
These just return my ultrawide fullscreen resolution, while I'm in a tiny window no less...

Reply:
oh, is there a good way of getting the screen ratio then? i know the game forces 16:9 but still fills strange to hardcode those values

Reply:
16/9 was written in so many places I had to abandon the ultrawide mod